### PR TITLE
Issue 2

### LINKED ISSUE #2: Fix the Difference method.
There is currently a method called `difference` that lives in `/lib/difference.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test difference` results in tests passing.

#### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request!

### PR DESCRIPTION
**Pull request by hugoduraes**
Fixes #2

### CODE REVIEW

File: test/square.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+test('0 squared to equal 0', () => {
+  expect(square(0)).toBe(0);
+});
+
+test('1 squared to equal 1', () => {
+  expect(square(1)).toBe(1);
+});
+
+test('3 squared to equal 9', () => {
+  expect(square(1)).toBe(1);
```
Comment (BlakeGuilloud) [inline test/square.test.js]: @hugoduraes I think this is meant to be `(square(9)).toBe(3)`! Fix that and it's in!

Comment (hugoduraes) [reply test/square.test.js]: Damn you copy paste! :(

Comment (BlakeGuilloud) [reply test/square.test.js]: haha it happens!

Comment (hugoduraes) [reply test/square.test.js]: Fixed now!

**Pull request merged**